RELEASE-NOTES.txt

```diff
@@ -2,6 +2,8 @@
 
 19.7
 -----
+* [*] Editor: Fix issue that caused previews on atomic sites to sometimes be out-of-date. [https://github.com/wordpress-mobile/WordPress-Android/pull/16332]
+
```

Comment:
Not sure why this extra space is here.. maybe introduced while git was auto-resolving conflicts?

Reply:
You're probably right. Thanks for catching it. Made sure there wasn't an extra space when I moved the release note from 19.7 to 19.8.